source/options.css

```diff
@@ -16,6 +16,14 @@ ul {
 	padding-left: 1.5em;
 }
 
+ul:not(.invalid) li.valid {
+	list-style: '✔️ ';
```

Comment:
Unfortunately Safari doesn't support custom strings in `list-style`, so this needs to be reworked with `::before` instead

Reply:
ok.

I dont have safari so I will do it and have you test it

Reply:
There's no need. `::before` works everywhere. Just replace it and it's fine